discordbot/bot/dictionary-module/dictionary.go

```diff
@@ -31,6 +31,13 @@ func (m *Module) Register(ctx context.Context, session *discordgo.Session) error
 
 func (m *Module) define(ctx commandContext, word string) error {
 	word = strings.TrimSpace(strings.ToLower(word))
+
+	initError := m.Client.GetInitError()
+	if m.Client.GetInitError() == nil {
```

Comment:
This condition should be reversed. If the error is nil then initialization is fine. Also, use `initError` variable since you have it?